### PR TITLE
Feat: make it possible to increase MaxSessionLifetime without increasing the SessionResult lifetime

### PR DESCRIPTION
**Pull request by ivard**
Changed default to 15 minutes, while keeping the SessionResult lifetime to 5 minutes.

### CODE REVIEW

File: irma/cmd/server.go [hunk 1]
```diff
@@ -115,7 +115,8 @@ func setFlags(cmd *cobra.Command, production bool) error {
 	flags.StringSlice("revoke-perms", nil, "list of credentials that all requestors may revoke")
 	flags.Bool("skip-private-keys-check", false, "whether or not to skip checking whether the private keys that requestors have permission for using are present in the configuration")
 	flags.String("static-sessions", "", "preconfigured static sessions (in JSON)")
-	flags.Int("max-session-lifetime", 5, "maximum duration of a session once a client connects in minutes")
+	flags.Int("max-session-lifetime", 15, "maximum duration of a session once a client connects in minutes")
```
Comment (ivard) [inline irma/cmd/server.go:118]: This means the default value of `ClientTimeout` also increases to 15 minutes. I'll check whether that's okay.

Comment (bobhageman) [reply irma/cmd/server.go:118]: @ivard, to be sure: has it been found ok?

Comment (ivard) [reply irma/cmd/server.go:118]: Yes, @MartijnYivi did not see any blocking issues. But if you have a strong opinion about this, then feel free to say so.

Comment (bobhageman) [reply irma/cmd/server.go:118]: Nope, think it is just fine, just double checking!;)

**Review by bobhageman — APPROVED** [review]

**Review by sjoerdlmkns — APPROVED** [review]

**Review by sanderhollaar — APPROVED** [review]

**Pull request merged**